cmd/src/batch.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+)
+
+var batchCommands commander
+
+func init() {
+	usage := `'src batch' manages batch changes on a Sourcegraph instance.
+
+Usage:
+
+	src batch command [command options]
+
+The commands are:
+
+	apply                 applies a batch spec to create or update a batch
+						  change
```

Comment:
```suggestion
	apply                 applies a batch spec to create or update a batch change
```

Reply:
It has to be wrapped because it goes past 80 characters, but I messed up the whitespace preceding the word, so I'll fix that.